### ISSUE TITLE
iframe embeds lack focus indicator

### ISSUE THREAD

**Issue by clhenrick** [clhenrick/clhenrick.io#124]
Example violation in `/work/from-coast-to-coast/`:

<img width="2566" alt="Screenshot 2024-12-31 at 12 19 27 PM" src="https://github.com/user-attachments/assets/0430c42a-06af-4df8-a20e-797af13d1096" />

Tasks:

- [x] visible focus indicator on iframe contents
- [x] iframe should have a valid `title` attribute (currently missing) to give it an accName
- [x] use SVG icon for "play" button to support forced colors mode
- [x] make sure link to load video player has an accName

**Comment by clhenrick** [comment]
https://codepen.io/clhenrick/pen/emOyWMQ?editors=1100
